DiceKitTests/AdditionExpression_Tests.swift

```diff
@@ -98,3 +98,57 @@ extension AdditionExpression_Tests {
     }
     
 }
+
+// MARK: - Operators
+// To do: Make these SwiftChecks
+extension AdditionExpression_Tests {
+    
+    func test_operator_shouldNotBreakIntAndInt() {
+        let notAnExpression = 21 + 21
+        
+        expect(notAnExpression) == 42
+    }
+    
+    func test_operator_shouldWorkWithIntAndDie() {
+        property("Int + Die") <- forAll {
+            (a: Int, b: Int) in
+            
+            let constant = a
+            let die = d(b)
+        
+            let expression = constant + die
+        
+            return expression.leftAddend == constant && expression.rightAddend == die
+        }
+    }
+    
+    func test_operator_shouldWorkWithDieAndInt() {
```

Comment:
This one can be rolled into the one above it. Just setup a die and constant for both `a` and `b`, then get two expressions and very both of them work as expected. That way you verify that you can add from either direction.